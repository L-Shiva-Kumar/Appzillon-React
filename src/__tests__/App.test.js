import { describe, expect, it } from '@jest/globals';
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import App from '../App';

test('should render a div saying React App', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
})

describe('<App />', () => {
    it('renders App Component', () => {
        const wrapper = shallow(<App />);
        expect(wrapper).toMatchSnapshot();
    });
})
