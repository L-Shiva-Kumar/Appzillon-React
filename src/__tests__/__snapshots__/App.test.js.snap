// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders App Component 1`] = `
<div>
  This is a react app!
</div>
`;

exports[`should render a div saying React App 1`] = `
<div>
  This is a react app!
</div>
`;
